api/machines.go

```diff
@@ -8,5 +8,6 @@ type Machines interface {
 	Get() ([]entity.Machine, error)
 	Create(machineParams *entity.MachineParams, powerParams map[string]string) (*entity.Machine, error)
 	Allocate(params *entity.MachineAllocateParams) (*entity.Machine, error)
+	AcceptAll() error
```

Comment:
`accept_all` is intended to add machines into MAAS, possibly by a non-admin user, and then later have an admin user approve these machines. Why pull this in for RAID?

Reply:
We,ve use it in automation scenarios, without admin or user intervention, from host discovery to setup custom image and push host to production, so accept_all was more fast approach in comparison with iteration through list of machines and acceptance only New one.
Probably I could separate it to separate PR if it looks like unsuitable api call for gomaasclient

Reply:
Yeah, I think it's a fine addition, just ideal to have it as a separate PR since it is not RAID related.

Reply:
Removed `accept_all` from this branch
Will create separate PR later